.devcontainer/devcontainer.json

```diff
@@ -1,9 +1,10 @@
 {
+	"name": "Semantic Kernel",
 	"image": "mcr.microsoft.com/devcontainers/universal:2",
 	"features": {
 		"ghcr.io/devcontainers/features/node:1": {},
 		"ghcr.io/devcontainers/features/dotnet:1": {
-			"version": "8"
+			"version": "9"
```

Comment:
We are happy to add STS 9 as an additional sanity check, any 9 related instrumentation should be additional to existing 8. Giving that I would suggest adding both 8, and 9 containers as possibilities, and sticking with the 8 when not possible.